src/Sentry/Laravel/Features/ConsoleSchedulingIntegration.php

```diff
@@ -266,7 +266,7 @@ private function createCheckIn(string $slug, CheckInStatus $status, ?string $id
         $options = SentrySdk::getCurrentHub()->getClient()->getOptions();
 
         return new CheckIn(
-            $slug,
+            Str::limit($slug, 128, ''),
```

Comment:
What about raising an exception here, if the `$slug` is `> 128` characters and cut of the string only, if the slug is autogenerated from a class (in `makeSlugForScheduled`)?

Reply:
We could. But that would make the code a lot more complex (to keep track where the name came from). I'm not sure there is much benefit to this. Especially since there is a very high chance the truncated string is still unique enough unless you are actively trying to break this.

Reply:
So I remembered another reason, this is why the current only exception we throw is where it is. That if we do it here we potentially disrupt only `schedule:run` which you probably only notice after deployment instead of before.

So for now I think it's best we leave it like this and if problems arise with the limiting go the string we will revisit this and refactor to make this an exception. But I think the risk is very minimal (we didn't have any limit before and just dropped the checkin probably, so this can be seen as an improvement even).